internal/cli/capact/cert_manager.go

```diff
@@ -0,0 +1,45 @@
+package capact
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	certv1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	certmanager "github.com/jetstack/cert-manager/pkg/client/clientset/versioned/typed/certmanager/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/retry"
+)
+
+// ApplyClusterIssuer creates or, if it already exists, updates a ClusterIssuer for cert-manager.
+// TODO: ensure issuer is ready.
```

Comment:
Do we want to address that now or later? Maybe it would be good to not have TODO comments without issue assigned 🤔

Reply:
hm.. currently we have 52 TODOs in our code, where almost all don't have issue assigned. IMO it's not needed as it's a low-hanging fruit, if someone will be here and have more time he can add it. Additionally, it serves as an info that we don't have such thing but it's nice to add some day.

In this case, I addressed it also on this PR but we need to decided about our TODO comments approach